certora/confs/LastUpdated.conf

```diff
@@ -3,12 +3,9 @@
         "lib/morpho-blue/certora/harness/MorphoHarness.sol",
         "certora/helpers/MetaMorphoHarness.sol",
     ],
-    "parametric_contracts": [
-        "MetaMorphoHarness",
-    ],
```

Comment:
This is to decide from which contract to take the set of functions for invariants. Previously, it was specified that invariants were checked to stay true for every functions of `MetaMorphoHarness`. Without this setting, any function from any contract is checked. In theory, it is enough to only check functions of the current contract, because those are the only functions able to change the storage. But there are case where you want to check other contracts as well. In particular when you have invariants that talk about the storage of multiple contracts, or when you have ghost variables that are updated from another contract

Reply:
So here it's safer to remove this setting